triggers/daml/Daml/Trigger/Internal.daml

```diff
@@ -67,40 +70,39 @@ adjust k f m =
 
 insertTpl : AnyContractId -> AnyTemplate -> ACS -> ACS
 #ifdef DAML_GENMAP && DAML_GENERIC_COMPARISON
-insertTpl cid tpl acs = acs { activeContracts = adjust cid.templateId addct acs.activeContracts }
-  where addct ol = Some $ (cid, tpl) :: fromOptional [] ol
+insertTpl cid tpl acs = acs { activeContracts = alter cid.templateId addct acs.activeContracts }
+  where addct = Some . GMap.insert cid tpl . fromOptional mempty
 #else
 insertTpl cid tpl acs = acs { activeContracts = (cid, tpl) :: acs.activeContracts }
 #endif
 
 #ifdef DAML_GENMAP && DAML_GENERIC_COMPARISON
-groupActiveContracts : [(AnyContractId, AnyTemplate)] -> GMap.Map TemplateTypeRep [(AnyContractId, AnyTemplate)]
-groupActiveContracts = foldr (\v@(cid, _) -> adjust cid.templateId (addct v)) GMap.empty
-  where addct v ol = Some $ v :: fromOptional [] ol
+groupActiveContracts : [(AnyContractId, AnyTemplate)]
```

Comment:
nit: I’m generally in favor of not making indentation depend on identifier length so a line-break before the start of the type signature.

Reply:
No problem. e58ffae6ec